recipes/nvtx/all/conanfile.py

```diff
@@ -35,8 +35,12 @@ def build(self):
     def package(self):
         copy(self, "LICENSE.txt", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
         copy(self, "*.h", src=os.path.join(self.source_folder, "c", "include"), dst=os.path.join(self.package_folder, "include"))
+        copy(self, "*.hpp", src=os.path.join(self.source_folder, "c", "include"), dst=os.path.join(self.package_folder, "include"))
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "nvtx3")
+        self.cpp_info.set_property("cmake_target_name", "nvtx3::nvtx3-c")
+        self.cpp_info.set_property("cmake_target_aliases", ["nvtx3::nvtx3-cpp", "CUDA::nvtx3"])
```

Comment:
The issue with these changes it that they would break existing consumers for the old versions too, just as you needed to update the existing test_package CMakeLists.

While I see that the file names are in fact wrong, I think we need to keep the actual `nvtx` one until at least the next major release, where breaking changes like this can be more acceptable.

What we can fix for now is the missing target names, by adding aliases to all of the missing ones, so keeping this only for now would make sense (as then current users keep their existing targets working too)


```suggestion
        self.cpp_info.set_property("cmake_target_aliases", ["nvtx3::nvtx3-cpp", "nvtx3::nvtx3-c"])
```

Note that I have not added the CUDA one, because I can't see where upstream is defining such a target, so I'd be happy to get some pointers there, thanks :)